continue-reading.html

```diff
@@ -4,10 +4,141 @@
     <title>Continue Reading</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="/styles/styles.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src="https://use.fontawesome.com/ca6c48c124.js"></script>
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+    <section id="header">
+        <h1 id="page_heading"><a href="index.html">Continue reading page</a></h1>
+        <h3 id="heading_subtext">A page for continuing and reading.</h3>
+        <div id="divider_container">
+          <div id="header_divider" class="divider"></div>
+        </div>

```

Comment:
You could have a little here by removing the divider container and putting the margin on the bottom of the heading_subtext h3.